README.md

```diff
@@ -63,6 +63,9 @@ config.omniauth :openid_connect, {
 | uid_field                    | The field of the user info response to be used as a unique id                                                                                                 | no       | 'sub'                      | "sub", "preferred_username"                         |
 | extra_authorize_params       | A hash of extra fixed parameters that will be merged to the authorization request                                                                             | no       | Hash                       | {"tenant" => "common"}                              |
 | allow_authorize_params       | A list of allowed dynamic parameters that will be merged to the authorization request                                                                         | no       | Array                      | [:screen_name]                                      |
+| pkce                         | Enable [PKCE flow](https://oauth.net/2/pkce/)                                                                                                                 | no       | false                      | one of: true, false                                 |
+| pkce_verifier                | Specify a custom PKCE verifier code.                                                                                                                          | no       | Random 32 character string | Proc.new { SecureRandom.hex(64) }                   |
+| pkce_options                 | Specify a custom implementation of the PKCE code challenge/method.                                                                                            | no       | Hash with default impl     | Proc to customise the code challenge generation     |
```

Comment:
`Hash with default impl` seems difficult to understand. Maybe we can just say `SHA256(code_challenge) in hex` if we use `Digest::SHA2.hexdigest(verifier)`?

Reply:
It's definitely simpler, I'm happy with that but I know other people are using this in production already so might affect them?